styleguide/samples/_forms.html.haml

```diff
@@ -0,0 +1,53 @@
+= render partial: "@citizensadvice/design-system/haml/header", local: { alt_language: alt_language }
+= render partial: "@citizensadvice/design-system/haml/navigation"
+.cads-grid-container
+  .cads-grid-row
+    .cads-grid-col-12
+      = render partial: "@citizensadvice/design-system/haml/breadcrumb", locals: {breadcrumbs: {"breadcrumb_links" => [{"url" => "/link", "title" => "Tables"}]}}
+%main.cads-main-content
+  .cads-grid-container
+    .cads-grid-row
+      .cads-grid-col-md-8
+        = render partial: "@citizensadvice/design-system/haml/notice_banner"
+    .cads-grid-row
+      .cads-grid-col-md-8
+        %h1 Consumer Query
+        %p From 1 April 2019 we are only able to provide general consumer advice to residents of England and Wales. If you are resident in Scotland and have a general consumer enquiry please contact Advice Direct Scotland / 0808 164 6000 
+        
```

Comment:
Sadly the build failed for a pedantic reason. You have some trailing whitespace at the end of these two lines that the 🤖  wants you to get rid of.

Reply:
The absolute embarassment. Think it's fixed now.

Reply:
Ugh now it's moaning about formatting in page.stories.js. Without getting you to set up a bunch of tools if you run `npx prettier --write styleguide/page.stories.js` in your terminal it should fix that. We might need to think about a way to have the CI tool tell you _all_ the code-style warnings in one go, rather than failing on the first thing.